projects/observability/src/shared/components/cartesian/d3/chart/cartesian-chart.ts

```diff
@@ -599,6 +601,12 @@ export class DefaultCartesianChart<TData> implements CartesianChart<TData> {
       this.tooltip.hide();
     }
 
+    this.eventListeners.forEach(listener => {
+      if (listener.event === ChartEvent.Hover) {
+        listener.onEvent(undefined);
```

Comment:
it's not intuitive that a mouse leave broadcasts a hover event with arg undefined - instead, create and broadcast a mouseleave event.